tasks/Makefile.hugo

```diff
@@ -10,6 +10,6 @@ hugo/run: components/build
 hugo/build: components/build
 	@[ "$(HUGO_PUBLISH_DIR)" != "/" ] || (echo "Invalid HUGO_PUBLISH_DIR=$(HUGO_PUBLISH_DIR)"; exit 1)
 	rm -rf $(HUGO_PUBLISH_DIR)
-	$(DOCKER_RUN) hugo --templateMetrics --stepAnalysis --config $(HUGO_CONFIG)
+	$(DOCKER_RUN) hugo --templateMetrics --trace trace.out --config $(HUGO_CONFIG)
```

Comment:
I think this will just write it to a file, which is `~/dev/null` if we don't output it =)